Analyzers/src/AnalyzerCore.C

```diff
@@ -719,6 +719,32 @@ std::vector<FatJet> AnalyzerCore::SmearSDMassFatJets(std::vector<FatJet> jets, i
 
 }
 
+//==== PDF AlphaS
+double AnalyzerCore::GetPDFAlphaSError(){
+
+  if(IsDATA) return 0.;
+  if(PDFWeights_AlphaS->size()!=2) return 0.;
+
+  return ( PDFWeights_AlphaS->at(1)-PDFWeights_AlphaS->at(0) ) / 2.;
+
+}
+//==== Renorm/Fact Scale
+double AnalyzerCore::GetScaleError(){
```

Comment:
This function on my previous commit was wrong.
For the scale errors, kinematics should be achieved using only one kind of scale shift and after that, envelope will be set as systematics bin by bin on histrogram, not choosing envelope event by event.
So, how about to make this function to be able to call six error with int 1~6 ?

Reply:
Hi SB, as suggested, related functions are now removed in the new commit.

Reply:
Good! How about to add example lines on ExampleRun.C for users?

Reply:
Hi SB, this is also added!

Reply:
Great! I will resolve conversation.